src/components/PasswordDialog.vue

```diff
@@ -100,9 +97,11 @@ export default Vue.extend({
 
 			const url = generateUrl('/login/confirm')
 			try {
-				const { data } = await axios.post(url, { password: this.password })
-				window.nc_lastLogin = data.lastLogin
-				this.$emit('confirmed')
+				if (this.password !== '') {
```

Comment:
Is it necessary to check the password content here if we disable the button?

Reply:
to be sure i would leave it, yes

Reply:
> Is it necessary to check the password content here if we disable the button?

It is not a form, though it should and in a form, we could submit a form with Enter even with a disabled button.